examples/deadline/All-In-AWS-Infrastructure-Basic/ts/lib/service-tier.ts

```diff
@@ -79,14 +79,26 @@ export interface ServiceTierProps extends cdk.StackProps {
 
   /**
    * Version of Deadline to use.
-   * @default The latest available release of Deadline is used
+   * @default - The latest available release of Deadline is used
    */
   readonly deadlineVersion?: string;
 
   /**
    * Whether the AWS Thinkbox End-User License Agreement is accepted or not
    */
   readonly acceptAwsThinkboxEula: AwsThinkboxEulaAcceptance;
+
+  /**
+   * Whether to enable secret management.
+   * @default - The secret management will be enabled
+   */
+   readonly enableSecretManagement?: boolean;
```

Comment:
This doesn't need to be optional since the value isn't optional in the config. We don't have it as optional in the python app and we should try to keep them as close to identical as possible.